assets/words/medium.txt

```diff
@@ -0,0 +1,100 @@
+Delighted
+Puppy
+Feline
+Bright
+Cobalt
+Leaves
+Aviary
+Dash
+Bound
+Newborn
+Pastry
+Aquatic
+Blooming
+Slide
+Story
+Sphere
+Sing
+Cap
+Toy
+Entertain
+Crib
+Celestial
+Lunar
+Marine
+Bear
+Quack
+Crimson
+Emerald
+Vessel
+Fruit
+Beverage
+Car
+Train
+Melody
+Grin
+Chuckle
+Slumber
+Rain
+Playground
+Rodent
+Amphibian
+Butterfly
+Companion
+Dwelling
+Entrance
+Lantern
+Cup
+Shore
+Verdant
+Chair
+Clock
+Pie
+Cream
+Game
+Candy
+Blossom
+Elephant
+Giraffe
+Lion
+Monkey
+Tiger
+Pig
+Horse
+Cow
+Sheep
+Rabbit
+Snake
+Fox
+Dragon
+Castle
+King
+Queen
+Prince
+Princess
+Wizard
+Magic
+Wand
+Sword
+Knight
+Shield
+Forest
+Mountain
+River
+Ocean
+Island
+Cave
```

Comment:
Interesting how this is medium

Reply:
I didn't pay too much attention to the words. These are all just randomly generated. I will definitely groom them a little bit